api/nbaData.js

```diff
@@ -74,7 +75,34 @@ const getPlayerRoster = function (accessToken, leagueId, teamId) {
     })
 }
 
+const getTeamIdByName = function (accessToken, leagueId) {
```

Comment:
If we require teamName in this call can we return teamId?
Or let's rename this to getAllTeams() or getLeagueStandings() and create a getTeamIdByName util function in yahooUtils.
What do you think?